proposal (copy).md

```diff
@@ -0,0 +1,139 @@
+---
+title: Pandas
+bibliography: biblio.bib
+link-citations: true
+---
+
+## Goals
+
+Fund the continued maintenance and improvements for Pandas
+(@mckinney-proc-scipy-2010), a Python library providing high-performance,
+easy-to-use data structures. Pandas is a foundational library in the Scientific
+Python Ecosystem, providing the most widely-used data structure for
+heterogenous, tabular data ().
+
+### Library Maintenance
+
+Pandas is a large library with many users. We have many open issues (about
+3,000) and Pull Requests (about 100). Keeping up with the stream of updates is
+time consuming, and as such we would like to fund time dedicated specifically
+for maintenance. This would include
+
+1. Promptly triaging newly opened issues
+2. Promptly reviewing new pull requests
+3. Ensuring issue conversations are progressing
+4. Ensuring that Pull Requests are not going stale and being abandoned
+
```

Comment:
Maybe we can add a bullet point here on mentoring / putting specific effort in reviewing PRs from certain contributors from a diversity perspective? (unless we would add a separate section about that)

Reply:
Done in #4